packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -404,7 +410,9 @@ class HorizontalOverflow extends Foundation<
      * onLoad handler to make sure any children affecting height are accounted for
      */
     private itemsOnLoad = (): void => {
-        const itemsHeight: number = this.getItemMaxHeight();
+        const itemsHeight: number = this.props.fixedHeight
```

Comment:
You might want to do a `typeof this.props.fixedHeight === "number"` check rather than a truthy check since explicitly passing 0 will enter the fail case. I'm not sure why you *would* pass 0, but I think we should support it

Reply:
Very good point, will update that